### DISCUSSION TITLE
Self-Hosted User Management

### DISCUSSION THREAD

**Discussion by SamGoody** [plausible/analytics#1401] [Self-Hosted Support] [answered]
I am running self hosted, and would like to edit the details of an existing user. 

For example, adding them as a viewer to an existing domain, or changing their email address.

How do I do that?

**Comment by metmarkosaric** [comment, marked as answer]
this is not possible in self-hosted. self-hosted is a long-term release so latest features won’t be immediately available as they’re battled-tested in the cloud before released to self-hosted.

**Comment by SamGoody** [comment]
Thanks for the quick answer.

I was able to create a new user as described in https://github.com/plausible/analytics/discussions/454, however I could not add him to the analytics of the site we are tracking.

If he cannot be added to an existing site during signup, and also not afterwards, than that means there is no real way to add a second user. 

Is that correct?
Is multiple user in the next LTS?

**Reply by metmarkosaric** [reply]
you can perhaps hack your way to it but it's not supported officially. official support for team members should be part of the next release which should happen over the next few weeks hopefully

**Reply by SamGoody** [reply]
Awesome! Will check back in a another month.

(I was thinking that if it would help, I would fork the docs and add instructions, but making multi user officially part of the GUI is even better)

Thanks for the product and for the support!
